CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### BugFixes
+
+- **Service Overview**
+  - search services field should allow speacial chars, digits
```

Comment:
Please put changelog update in PR descriptions to avoid conflicts

@manojava-gk @lavanya-bmw @kunalgaurav-bmw Please review

Reply:
Updated!

Reply:
> Updated!

Please remove CHANGELOG.md changes as @evegufy requested

Reply:
Done, Please check @lavanya-bmw